helm/cert-manager-app/charts/giantswarm-cluster-issuer/templates/cm.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ .Values.name }}
   namespace: {{ .Release.Namespace }}
   annotations:
-    "helm.sh/hook": "post-install"
+    "helm.sh/hook": "post-install,post-upgrade"
```

Comment:
The `post-upgrade` hook will be removed in a future release - it's to ensure that the self-signed clusterissuer is created. The existing letsencrypt clusterissuer will remain untouched.